src/overlays/actors/ovl_En_Dodojr/z_en_dodojr.c

```diff
@@ -467,55 +468,57 @@ void func_809F768C(EnDodojr* this, PlayState* play) {
     } else {
         Audio_PlayActorSound2(&this->actor, NA_SE_EN_DODO_K_DRINK);
         Actor_Kill(this->bomb);
-        this->timer3 = 24;
-        this->unk_1FC = 0;
-        this->actionFunc = func_809F773C;
+        this->timer = 24;
+        this->counter = 0;
+        this->actionFunc = EnDodojr_SwallowBomb;
     }
 }
 
-void func_809F773C(EnDodojr* this, PlayState* play) {
-    if (DECR(this->timer3) == 0) {
-        func_809F64D0(this);
+void EnDodojr_SwallowBomb(EnDodojr* this, PlayState* play) {
+    if (DECR(this->timer) == 0) {
+        EnDodojr_SetupSwallowedBombEffects(this);
         this->actor.flags &= ~ACTOR_FLAG_0;
-        func_809F6A20(this);
-        this->actionFunc = func_809F77AC;
+        EnDodojr_SetupFlipBounce(this);
```

Comment:
I know `EnDodojr_SetupSwallowedBombEffects` just got its name but now this is two SetupX in a row, `EnDodojr_SetupSwallowedBombEffects` is not a setup? idk I haven't seen "multi-setup-functions" usage elsewhere but this actor has weird patterns anyway

Reply:
yea i wouldnt say its a setup function. im also not sure why mzx said it has to start with a verb in a previous comment, its not an action either really. it just plays a sound effect and sets the color filter to make it blink

Reply:
idk how to name it, but let's try to help kentonm lol, thoughts on `EnDodojr_DoSwallowedBombEffects` ?

Reply:
`EnDodojr_DoSwallowedBombEffects` !

Reply:
changed to `EnDodojr_DoSwallowedBombEffects`. So sorry about the delay I completely forgot about this PR